docs/changes/coordinates/15220.api.rst

```diff
@@ -0,0 +1,4 @@
+
+Angle-related classes and functions have been consolidated into the module ``angles``.
+In particular, ``angle_utilties`` have been moved to ``angles.util``,
+and select errors have been moved to ``angles.errors``.
```

Comment:
If the users are supposed to import everything from `astropy.coordinates` then I don't see why we would even want to mention `angles`, let alone `angles.utils` or `angles.errors`. We should just state that there has been some refactoring that might break some imports from some modules within `astropy.coordinates`, but everything that could be imported from `astropy.coordinates` itself still can be, and what cannot be imported from `astropy.coordinates` is and has been considered private.

Reply:
Ok, you're leaning towards "private" module (from opening comment). Happy to do that as well.

Reply:
Agreed with @eerovaher - let's stick with the current astropy scheme where users are supposed to import only from submodules, not below. Arguably, this is just a refactoring that does not even need a changelog entry. But I do think it is a good idea to have one anyway, if we phrase it along the lines @eerovaher suggested. It is also fine to be friendly and note that if people use stuff that is not exposed on top, they should feel welcome to submit a feature request to make it public -- since then obviously it is useful!

Reply:
Done.